client/components/paginated-survey/components/question-step.tsx

```diff
@@ -0,0 +1,66 @@
+import { StepContainer } from '@automattic/onboarding';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { Question, QuestionType } from '../types';
+import QuestionMultipleOptions from './question-multiple-selection';
+import QuestionSingleOption from './question-single-selection';
+import './style.scss';
+
+const questionsMap = {
+	[ QuestionType.SINGLE_CHOICE ]: QuestionSingleOption,
+	[ QuestionType.MULTIPLE_CHOICE ]: QuestionMultipleOptions,
+};
+
+export type QuestionSelectionType = {
+	question: Question;
+	onChange: ( questionKey: string, value: string[] ) => void;
```

Comment:
If there is a need for default selected value for a question, this could be extended with it, in the future of course (this just popped into my mind so I just wanted to write it down).

Reply:
Yes, indeed. I'll pass the value selected as a prop.